cmd2/cmd2.py

```diff
@@ -402,13 +403,15 @@ def __init__(self, completekey: str = 'tab', stdin=None, stdout=None, persistent
         self.hidden_commands = ['eof', 'eos', '_relative_load']
 
         # Commands to exclude from the history command
+        # initialize history
+        self.persistent_history_length = persistent_history_length
```

Comment:
Documentation for `persistent_history_file` and `persistent_history_length` should be updated in the `cmd2.Cmd.__init__` docstring.

Also, documentation on "Searchable command history" should be mildly updated in the Sphinx docs.